pom.xml

```
@@ -290,6 +290,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>cobertura-maven-plugin</artifactId>
+                <version>2.7</version>
```

Shall we parameterize this version as we did in other plugins?
```suggestion
                <version>2.7</version>
```